### PR TITLE
Move from PhantomJS to a Selenium grid

### PR DESCRIPTION
**Pull request by ylavoie**
This PR departs from PhantomJS to the profit of a Selenium Grid. Chrome, Firefox or Opera can be used.
It also maps the host user to the working user in the `lsmb` container and bind mount the home directory so that the user is defined.
Finally, it maps the proxy server to 5000 instead of 80 to remove conflicts with existing installations.

### CODE REVIEW

**Comment by ehuelsmann** [comment]
Great work! One glitch:

```plain
$ ../ldd/lsmb-dev 18 up -d
Creating network "ld18_internal" with driver "bridge"
Creating ld18_db_1       ... done
Creating ld18_mailhog_1  ... done
Creating ld18_selenium_1 ... done
Creating ld18_chrome_1   ... done
Creating ld18_chrome_2   ... done
Creating ld18_chrome_3   ... done
Creating ld18_chrome_4   ... done
Creating ld18_chrome_5   ... done
Creating ld18_lsmb_1     ... done
Creating ld18_proxy_1    ... done

======================================
== LedgerSMB '1.8-oldcode-entrypoints'
== should be available at
======================================
http://<no value>
======================================
```

Before, it used to report the IP address of the container (on the default network, I presume), but that's no longer happening...

**Comment by ehuelsmann** [comment]
There's another glitch. I run a lot of docker compose instances on the same machine at the same time (each on their own branch); doing so with this change applied, results - for the second container-set - in:

```plain
 ../ldd/lsmb-dev master up -d
Creating network "ldmaster_internal" with driver "bridge"
Creating ldmaster_db_1 ... 
Creating ldmaster_mailhog_1 ... 
Creating ldmaster_selenium_1 ... 
Creating ldmaster_selenium_1 ... error
Creating ldmaster_db_1       ... done
Creating ldmaster_mailhog_1  ... done
90d0cd2b3fe196d0ddf5d8a): Bind for 0.0.0.0:4444 failed: port is already allocated

ERROR: for selenium  Cannot start service selenium: driver failed programming external connectivity on endpoint ldmaster_selenium_1 (6e3ea0e92fae40b1fc6b708e03dca3a0d5b8b16f490d0cd2b3fe196d0ddf5d8a): Bind for 0.0.0.0:4444 failed: port is already allocated
ERROR: Encountered errors while bringing up the project.
Error: No such object: ldmaster_proxy_1

======================================

Error: No such container: ldmaster_lsmb_1

======================================
```

**Comment by ylavoie** [comment]
> Before, it used to report the IP address of the container (on the default network, I presume), but that's no longer happening...

If and when there was only 1 network involved. I'll fix

**Comment by ylavoie** [comment]
> 
> Before, it used to report the IP address of the container (on the default network, I presume), but that's no longer happening...

Fixed to:
```
======================================
== LedgerSMB 'master'
== should be available at
======================================
host:   http://host:49169
proxy: http://172.19.0.3
psgi:   http://172.19.0.2:5762
======================================
```

File: README.md [hunk 1]
```diff
@@ -77,13 +85,13 @@ http://172.20.0.6/login.pl allows to log into the company.
 
 Three commands exist to run tests:
 
-* `make test`  
+* `make test`
```
Comment (ehuelsmann) [inline README.md:88]: The two spaces at the end force a line break, which was intentional. Does it need to go?

Comment (ylavoie) [reply README.md]: Trailing spaces are always removed. A trailing `\` does the same

**Pull request merged**